docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -139,9 +139,18 @@ Currently there is no option to select an alternative region for storage.
 
 ## Will Teleport be automatically upgraded with each release?
 
-The Teleport control plane is routinely upgraded within 1-2 weeks of a Teleport Core or Enterprise Release.
-New features may not be available to cloud users until compatibility is tested and fixed, and some features may not work in the cloud environment.
-Individual customers may not be upgraded to the latest release if they are running agents that are outside of the supported compatibility of the control plane.
+Teleport Cloud follows the Teleport [Production
+readiness](../../preview/upcoming-releases.mdx) guidelines.
+
+- Teleport Cloud control plane major version upgrades occur after the first
+  minor release (for example (=teleport.major_version=).1.0). This typically
+  happens within one month of a new major version release becoming available.
+- After the control plane has been upgraded to the first major version, minor
+  and patch release upgrades typically occur within a week.
+
+Individual customers may not be upgraded to the latest release if they are
```

Comment:
Are you sure we want to add this. Some may interpret as permission to never upgrade and expect that we will indefinitely hold back their control plane for them.

Reply:
Good point, removed it because we're soon not even going to offer this.